didcomm/protocols/forward/forward.py

```diff
@@ -71,13 +166,29 @@ async def unpack_forward(
 
     :return: Forward plaintext
     """
+    # TODO decrypt_by_all_keys ???
```

Comment:
Probably we need to pass `Optional[UnpackConfig]` as in `unpack` and get `decrypt_by_all_keys` from there.

Reply:
updated